rfcs/0005-managed-attributes-image-automation/README.md

```diff
@@ -0,0 +1,82 @@
+# RFC-0005 Managed attributes on Image Automation
+
+**Status:** provisional
+
+**Creation date:** 2021-12-16
+
+**Last update:** 2021-12-16
+
+## Summary
+
+Image automation controller can update some attributes of a kubernetes object. 
```

Comment:
Would you mind changing this to something that more directly explains what the RFC covers? Something like "Flux should allow referencing more container image metadata attributes in the image automation Setters strategy"

Reply:
I rewrote a little bit this part.